state/state.go

```diff
@@ -146,7 +146,7 @@ func (st *state) tryLoadLastInfo() error {
 
 	totalStake := int64(0)
 	st.store.IterateValidators(func(val *validator.Validator) (stop bool) {
-		totalStake += val.Stake()
+		totalStake += val.Power()
```

Comment:
Stake is the same as Power for all validators except for Bootstrap validators.

The network starts with 4 bootstrap validators which don't have any stake in advance. So their power sets to one. This helps consensus to start the bootstrapping phase.

Note that Zarb doesn't have any pre-minted or pre-allocated tokens.

Reply:
> Stake is the same as Power for all validators except for Bootstrap validators.
exactly for this they should be used cautiously and the they should be distinguished as result of this exception i passed a [test](https://github.com/zarbchain/zarb-go/pull/123/files/6eac6392c331973bd133346ed78df485e34e0537#diff-d16667d244a9c4e4211ee05f2d187488343c959396a63891f179ce5eaf438e04L64) `around` witch may raise issues along the way ,unlikely but possible

Reply:
TotalStake used for VRF and sortition. What is your error?

Reply:
> 
> 
> TotalStake used for VRF and sortition. What is your error?

it might be just test env but may happen in slow networks as well that the newly generated validators always have stake 1!

Reply:
> it might be just test env but may happen in slow networks as well that the newly generated validators always have stake 1

Stake for new validators is Zero. Power only applies inside consensus.